File: src/main/java/gregtechlite/gtlitecore/mixins/LateMixinLoader.java
package gregtechlite.gtlitecore.mixins;

import zone.rong.mixinbooter.ILateMixinLoader;

import java.util.List;

@SuppressWarnings("unused")
public final class LateMixinLoader implements CustomMixinLoader, ILateMixinLoader
{
    @Override
    public List<String> getMixinConfigs()
    {
        return createMixinConfigs("gregtech", "jei", "modularui");
    }
}

File: src/main/kotlin/gregtechlite/gtlitecore/mixins/LateMixinLoader.kt
package gregtechlite.gtlitecore.mixins

import zone.rong.mixinbooter.ILateMixinLoader

@Suppress("unused")
class LateMixinLoader : CustomMixinLoader, ILateMixinLoader
{
    override fun getMixinConfigs(): List<String> = createMixinConfigs("gregtech", "jei", "modularui")
}